mender/templates/tests/test-mender.yaml

```diff
@@ -0,0 +1,46 @@
+{{- if .Values.tests.enabled }}
+apiVersion: v1
+kind: Pod
+metadata:
+  name: {{ include "mender.fullname" . }}-test-mender
+  labels:
+    {{- include "mender.labels" . | nindent 4 }}
+  annotations:
+    "helm.sh/hook": test
+spec:
+  containers:
+    - name: wget
+      image: busybox
+      command: ['/bin/sh', '-c']
+      args:
+        {{- if and (.Values.global.enterprise) (.Values.tenantadm.enabled) (.Values.auditlogs.enabled) }}
```

Comment:
This condition is a bit weird, since `.Values.global.enterprise` only adds `auditlogs`, `devicemonitor` and `tenantadm`. The rest of the stack should be the same.

Reply:
I think we have to restructure the `enabled` services...